src/grimoirelab/core/scheduler/views.py

```diff
@@ -78,3 +80,68 @@ def add_task(request):
         'message': f"Task {task.id} added correctly"
     }
     return JsonResponse(response, safe=False)
+
+
+@require_http_methods(["POST"])
+@csrf_exempt
+def add_repository(request):
```

Comment:
For this iteration, let's say that the data about the repository/task can't be updated. Let's keep it as simple as possible. So, when `add_repository` tries to add a repository that already exists, return a **405 Method not allowed**.